tests/bootstrap_spec.lua

```diff
@@ -0,0 +1,34 @@
+local a = require "plenary.async_lib.tests"
+
+a.describe("initial start", function()
+  local uv = vim.loop
+  local home_dir = uv.os_homedir()
+  local lvim_config_path = home_dir .. "/.config/lvim"
+  local lvim_runtime_path = home_dir .. "/.local/share/lunarvim/lvim"
+  a.it("should not be reading default neovim directories in the home directoies", function()
```

Comment:
Check https://github.com/Olivine-Labs/luassert#luassert to work the asserters you are using in the file

Reply:
can you expand on that? do you mean that we should add extra asserts or what exactly? Isn't this library included with plenary anyway?

Reply:
For instance, `assert.same(found_illegal_path, false)` could be `assert.False(found_illegal_path)`
I would probably also switch `assert.is_true` with `assert.True`